src/Lykke.Job.TransactionHandler/Projections/OperationHistoryProjection.cs

```diff
@@ -204,7 +203,7 @@ public class OperationHistoryProjection
 
         public async Task Handle(CashoutTransactionStateSavedEvent evt)
         {
-            await _log.WriteInfoAsync(nameof(OperationHistoryProjection), nameof(CashoutTransactionStateSavedEvent), evt.ToJson(), "");
+            _log.WriteInfo(nameof(CashoutTransactionStateSavedEvent), evt.ToJson(), "");
```

Comment:
This is not needed with latest version of Lykke.Cqrs - such logging is done from the library